community/vendor-companies-engagement-procedure.md

```diff
@@ -0,0 +1,184 @@
+## Procedure for Engaging With Vendor Companies
+
+### Summary
+
+This document outlines the procedure for engaging with vendor companies, and
+how vendor companies can represent themselves and their interests in the SPIFFE
+community.
+
+The goal is to provide a clear and consistent process for engaging with 
+vendor companies that are included but not limited to those that use SPIFFE and 
+SPIRE, provide a proprietary SPIFFE-compatible service, or are interested in 
+contributing to the SPIFFE project.
+
+More specifically, this document includes guidelines for communication, 
+technical contact designation, and handling potential disruptive participants.
+
+### Contacts
+
+#### Objective
+
+To ensure effective communication and support for vendor’s product or service
+consumers using the SPIFFE specification by designating responsible technical 
+contacts.
+
+#### Guidelines
+
+##### Designation of Contacts
+
+* Each vendor company that directs their customers to the SPIFFE Slack channel 
+  must designate two contacts. At least one of these contacts are encouraged
+  (*but not required*) to be a technical contact. See the definition of roles
+  and responsibilities of technical and community contacts in the subsequent
+  **Kinds of Contacts** subsection.
```

Comment:
Missing blank line between paragraphs?
```suggestion
  **Kinds of Contacts** subsection.
  
```

Reply:
It's a (bit long…ish) bullet point, not a paragraph. Adding a new line would make it two separate bullet points with one item on each.